completion/pgffor.cwl

```diff
@@ -0,0 +1,9 @@
+# pgffor package
+# muzimuzhi, 8 Aug 2020, pgffor in tikz v3.1.5b
```

Comment:
why separate file ?
Is there any scenario where pgffor is separated from pgfcore ?

Reply:
Sometimes, yes. As the `pgfmanual` says in sec. 88,

> This package (`pgffor`) can be used independently of PGF. 

Personally I sometimes load `pgffor` without PGF/tikz, mostly when preparing small examples as an answer to online LaTeX questions. In the future I might add more lines to `pgffor.cwl` if possible. Will deeper dependency of cwl files have obvious impact on the user experience?

Reply:
Okay, this is fine